src/components/EpisodePreview/styles.scss

```diff
@@ -62,14 +62,135 @@
 		}
 	}
 
-	.content {
-		position: absolute; //placeholder
+	.container {
+		position: absolute; 
+
+		&[aria-hidden="false"]{
+			opacity: 1;
+			display: flex;
+			height: 100%;
+			width: 100%;
+			top: 0;
+			left: 0;
+			z-index: 3;
+			flex-direction: column;
+			align-items: center;
+			font-size: 1.62rem;
+			background-color: green;
+		}
 
 		&[aria-hidden="true"]{
 			opacity: 0;
+			pointer-events: none;
 		}
 	}
 
+		.header {
+			position: absolute;
+			top: 70%;
+			z-index: 2;
+		}
+
+			.episode, .title {
+				text-align: center;
+			}
+
+			.episode {
+				font-size: 1em;
+			}
+
+			.title {
+				font-size: 1.46em;
+			}
+
+			.controls {
```

Comment:
I wanted to do this but since the prev/next are selectively rendered, you end up with weird alignment for episode 1 and episode 10 (the ones where they only have one button).

Reply:
I've added the space between anyway as it is a nicer approach. Any suggestions for best way to handle episode 1 and 10?

Reply:
aha, here's the trick I was telling you about before - on 'previous', put

```
margin-left: 0;
margin-right: auto;
```
and on next put
```
margin-right: 0;
margin-left: auto;
```

It will do the same thing that it's currently doing, apart from it supports the edge case you mentioned there; the benefits of `space-between` over this approach are:

- brevity when you know you're always going to have 2 children
- when you have more than 2 children and you need to space them all evenly.

Reply:
Noice, good to know!